pylint/constants.py

```diff
@@ -43,6 +43,19 @@
 MSG_TYPES_LONG: dict[str, str] = {v: k for k, v in MSG_TYPES.items()}
 
 MSG_TYPES_STATUS = {"I": 0, "C": 16, "R": 8, "W": 4, "E": 2, "F": 1}
+PYLINT_MSGS = [
```

Comment:
Can we get this dynamically from the `PyLinter` class?

Reply:
Thanks @DanielNoord, I now understood the meaning of the `MSGS` constant after taking another look and got rid of this unnecessary additional constant.

Do you mind taking another look?